lib/utils/webpack.config.js

```diff
@@ -368,12 +368,12 @@ module.exports = (program, directory, suppliedStage, webpackPort = 1500, routes
         config.loader('less', {
           test: /\.less/,
           exclude: /\.module\.less$/,
-          loader: ExtractTextPlugin.extract(['css?minimize', 'less']),
+          loader: ExtractTextPlugin.extract(['css?minimize', 'postcss']),
         })
         config.loader('sass', {
           test: /\.(sass|scss)/,
```

Comment:
@kyleamathews This is just the normal sass/less. I left the module loaders untouched. Line 336 is from the 'develop' section.